### DISCUSSION TITLE
Copy link with only selection text

### DISCUSSION THREAD

**Discussion by Thejungleee** [jglev/obsidian-carry-forward#27] [Q&A]
Hello! 

First things first - amazing plugin! It has a huge potential, probably above my skills, hence I wanted to consult on one use case. Consider the text block below:

> Sed ut perspiciatis, unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam eaque ipsa, quae ab illo **inventore veritatis et quasi architecto beatae vitae dicta sunt, explicabo**. Nemo enim ipsam voluptatem, quia voluptas sit, aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos, qui ratione voluptatem sequi nesciunt, neque porro quisquam est, qui dolorem ipsum, quia dolor sit, amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt, ut labore et dolore magnam aliquam quaerat voluptatem.

I would like only to create the bold part of text with the link to the whole block, aka:

`
inventore veritatis et quasi architecto beatae vitae dicta sunt, explicabo (see {{Link}}). 
`

Could you please help me configure the plugin so I could achieve this? Tried to understand the regex but it's still looks too complicated for me to figure it out. Your help will be greatly appreciated! Thanks!

**Comment by jglev** [comment]
> First things first - amazing plugin!

Thank you for your words! : D

> Could you please help me configure the plugin so I could achieve this? Tried to understand the regex but it's still looks too complicated for me to figure it out. Your help will be greatly appreciated! Thanks!

Thank you for writing this up! This turned out to be a bug; the plugin is always meant to copy just the selection (or the whole line only if there is no selection). I've released version [`1.4.4`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.4) just now, which I _think_ addresses it. Would you be willing to confirm whether the updated version behaves as you expect?

**Comment by Thejungleee** [comment]
hi Jacob! It's amazing - now it works like a charm, and solves plethora of
my notes problems! Thank you for your quick response and resolving the
issue! Your plugin is an amazing thing now! All the best for future
development!

sob., 17 gru 2022 o 18:21 Jacob Levernier ***@***.***>
napisał(a):

**Reply by jglev** [reply]
Great! Thank you for writing to confirm, and for reaching out! : D
